vendor/perfect-scrollbar/main.scss

```diff
@@ -0,0 +1,123 @@
+// Colors
+$ps-rail-hover: #eee;
+$ps-bar-default: #aaa;
+$ps-bar-hover: #999;
+
+// Helper mixins
+@mixin border-radius($r) {
+  -webkit-border-radius: $r;
+     -moz-border-radius: $r;
+      -ms-border-radius: $r;
+          border-radius: $r;
+}
+
+@mixin transition($t...) {
+  -webkit-transition: $t;
+     -moz-transition: $t;
+       -o-transition: $t;
```

Comment:
Line should be indented 2 spaces, but was indented 7 spaces<br>Avoid vendor prefixes.